packages/sliderInput/style.ts

```diff
@@ -0,0 +1,181 @@
+import { css } from "emotion";
+import {
+  spaceL,
+  spaceS,
+  themeBorder,
+  themeBgDisabled,
+  themeBgPrimary,
+  themeBrandPrimary,
+  themeError,
+  themeSuccess,
+  themeTextColorDisabled,
+  white
+} from "../design-tokens/build/js/designTokens";
+
+const sliderInputHeight = 14;
```

Comment:
Toggle input height is 16, wonder if they should be the same?

Reply:
The size was taken directly from the invision design.

Reply:
@TattdCodeMonkey yeah best to keep same as mockup as control would be too big at same height but its box area should take up same space as other toggles so everything aligns when used together in a form layout.

Reply:
@danielone I don't think keeping it the same size as other toggles is possible. The checkbox is a square 16x16 and the radio button is a circle. This is an oval that is 14x28 and the width is needed to move the slider and display the check.